doc/man/bitcoin-qt.1

```diff
@@ -193,7 +193,7 @@ Accept connections from outside (default: 1 if no \fB\-proxy\fR or \fB\-connect\
 .HP
 \fB\-listenonion\fR
 .IP
-Automatically create Tor hidden service (default: 1)
+Automatically create Tor onion service (default: 1)
```

Comment:
The first line of each of the `man` files states: `DO NOT MODIFY THIS FILE!  It was generated by...`